tools/ci/run_tests.sh

```diff
@@ -19,7 +19,9 @@ PROJECT_DIR=$(cd "$(dirname "$0")/../.."; pwd)
 
 python -m pip -q install --upgrade "feathub-nightly[flink]"
 
-docker build -q --rm -t feathub-flink -f ./docker/Dockerfile .
+docker build -q --rm -t feathub-flink -f ./docker/Dockerfile.flink .
+
+docker build -q --rm -t feathub-spark:3.3.1 -f ./docker/Dockerfile.spark ./docker
```

Comment:
Since we don't have plan to support multiple Spark versions at this moment, it seems better not to tag the docker image with the version, for simplicity and consistency with the Flink docker image.